src/System.Windows.Extensions/src/System/Security/Cryptography/X509Certificates/SafeX509Handles.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+namespace System.Security.Cryptography.X509Certificates
+{
+    internal sealed class SafeCertContextHandle : SafeHandleZeroOrMinusOneIsInvalid
+    {
+        // 0 is an Invalid Handle
```

Comment:
This says 0 is invalid, but the safe handle is deriving from "ZeroOrMinuteOneIsInvalid"?